frontend/src/utils/hooks/sources.ts

```diff
@@ -34,7 +34,7 @@ export const useSources = (options: Options = defaultOptions, fetch = false): Li
   const fetchSources = (): void => {
     const url = `${api.routes.SOURCES}?limit=${options.limit}&offset=${options.offset}&search=${
       options.search || ''
-    }`;
+    }&frozen=2`;
```

Comment:
I saw a similar change on Chris' branch, so I expect a clash ... so either branch off his, or wait in line to resolve the conflict ... frankly, I'd have preferred if you used the API endpoint to fetch only exactly what you need, but I also don't think an API change was worth it in this case given how few frozen data sources we have & how we're planning on clearing them out every few months. So, go ahead with the filter but consider how Chris' changes work to eliminate the conflict.

Reply:
Rebased `develop`, now using Chris' version